src/js/components/Menu.js

```diff
@@ -39,6 +39,18 @@ var MenuDrop = React.createClass({
       down: this._onDownKeyPress
     };
     this.startListeningToKeyboard(this._keyboardHandlers);
+    var menuItems = this.refs.navContainer.getDOMNode().childNodes;
+    for (var i = 0; i < menuItems.length; i++) {
+      menuItems[i].setAttribute('role', 'menuitem');
+      
+      menuItems[i].setAttribute('id', menuItems[i].getAttribute('data-reactid'));
```

Comment:
What if the caller has already set an id attribute on the menu item? I wouldn't want Grommet to overwrite it.